filters.ml

```diff
@@ -1066,6 +1088,7 @@ let run com tctx main =
 			captured_vars com;
 			promote_complex_rhs com;
 			if com.config.pf_add_final_return then add_final_return else (fun e -> e);
+			if com.platform = Js then wrap_js_exceptions com else (fun e -> e);
```

Comment:
We have problems with this for python as well, so I'm fine with adding target-specific filters there. I don't think we have to abstract this in any way, so what you did there is fine.